policies.go

```diff
@@ -175,10 +183,22 @@ func (s *SimpleRetryPolicy) Attempt(q RetryableQuery) bool {
 	return q.Attempts() <= s.NumRetries
 }
 
+func (s *SimpleRetryPolicy) AttemptLWT(q RetryableQuery) bool {
+	return s.Attempt(q)
+}
+
 func (s *SimpleRetryPolicy) GetRetryType(err error) RetryType {
 	return RetryNextHost
 }
 
+// Retrying on a different host is fine for normal queries,
+// but will cause contention and timeouts if other clients
+// send statements touching the same partition to the original node
+// at the same time.
```

Comment:
```suggestion
// Retrying on a different host is fine for normal (non-LWT) queries,
// but in case of LWTs it will cause Paxos contention and possibly
// even timeouts if other clients send statements touching the same
// partition of the original node at the same time.
```

Reply:
>  touching the same partition in the original node at the same time.

Not sure what this means.

Reply:
it should be "if other clients send statements touching the same partition to the original node at the same time."

Reply:
`other clients` - other than ours?
`partition to the original node` - what is the original node? Primary replica? What does it mean "partition to ... node" ?

Reply:
The contention happes if we send a statement to node Y (in case of retry policy: we originally sent it to node X) and another client send the statement that touches the same partition to node X ("original node" in this sentence)

Reply:
> `other clients` - other than ours? 
What other meaning could it possible have?


>  `partition to the original node` - what is the original node? 

This is a comment in retry policy. Original node is the node that we originally sent the statement to.

> What does it mean "partition to ... node" ?

There is a "send" verb before this.

"send statements touching the same partition to the original node"

They are sending statements.
Statements that touch the same partition as the statement we are sending.
They are sending it to the node we originally sent it to.

Reply:
Got it, updating the suggestion.

Reply:
```suggestion
// Retrying on a different host is fine for normal (non-LWT) queries,
// but in case of LWTs it will cause Paxos contention and possibly
// even timeouts if other clients send statements touching the same
// partition to the original node at the same time.
```